app/lib/efile/ny/it201.rb

```diff
@@ -329,7 +329,7 @@ def calculate_line_72
         # NY income tax withheld
         # sum of tax withheld from all income documents
         @direct_file_data.total_w2_state_tax_withheld +
-          @intake.state_file1099_gs.sum(&:state_income_tax_withheld)
+          @intake.state_file1099_gs.sum(&:state_income_tax_withheld_amount).round
```

Comment:
same here, do we round after sum or round then sum? does this make no difference? I think it would make a difference

i.e. 

amount1: 50.5
amount2: 50.7

amount1.round + amount2.round = 102
amount1 + amount2 = (101.2).round = 101

Reply:
we should probably round and then sum i think. thank you for the catch!